lib/beaker-hostgenerator/data.rb

```diff
@@ -51,5 +61,7 @@ module Data
         'consoleport' => 443,
       }
     }
+
+    module_function :pe_version, :pe_family, :pe_upgrade_version, :pe_upgrade_family
```

Comment:
I don't know, nothing in Ruby makes sense to me and whenever I think too hard about it I become immaturely angry with myself for working a job that requires me to write a language that I consider to be bad programming practice. (Things I tend to think of as good programming practice often turn out to be "ineloquent" in the duby community.)

On a more reasonable note, I may have done this back when I tried calling these functions when setting values for a module constant here:
https://github.com/puppetlabs/beaker-hostgenerator/blob/master/lib/beaker-hostgenerator/generator.rb#L14

As it turns out, the use of a module constant whose value ultimately is set programatically and can change in different circumstances is actually quite a bad idea (hopefully I've learned my lesson). If I dwell too much on this point I am fairly certain I will become irrationally angry that ruby allows module constants to be set with functions or even to values that can be mutated at all.

TL;DR I think the use of mixins is reasonable so I'll go ahead and do that, thanks for the feedback!